readme.md

```diff
@@ -116,7 +152,9 @@ How is rebase different than a merge?
 
 Your Answer:
 ```text
+Merge puts the latest commit on a branch onto the end of the branch being merged to.  The branch histories and commits remain separate.
 
+Rebasing merges the two branches entirely, so there is no longer a commit history in each branch.
```

Comment:
not quite... check out https://github.com/ga-dc/wdi7-quiz9/blob/solution/readme.md#question-9